drivers/rtc/Kconfig.ds3231

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2019 Peter Bigot Consulting, LLC
+# Copyright (c) 2024, Gergo Vari <work@varigergo.hu>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config RTC_DS3231
+	bool "Maxim DS3231 RTC/TCXO"
+	default y
+	depends on DT_HAS_MAXIM_DS3231_ENABLED
+	depends on I2C
+	depends on RTC
```

Comment:
`depends on RTC` This one is not required as this file is only included if `CONFIG_RTC=y`